### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pgarciaegido**
Adding wallet UI components information and references to the docs on the React SDK readme

Screenshot 2023-03-14 at 19 21 37

### CODE REVIEW

File: packages/ui/react-ui/README.md [hunk 1]
```diff
@@ -14,7 +14,20 @@ First, add the Crossmint Client SDK to your project with the following command:
 yarn add @crossmint/client-sdk-react-ui
 ```
 
-The next steps depend on which chain you're using for your project. For detailed integration instructions, visit our documentation for [Solana](https://docs.crossmint.io/accept-credit-cards/integration-guides/solana-candy-machine/b-i-have-an-existing-candy-machine-website/add-crossmint-for-solana-react.js-next.js), [Polygon](https://docs.crossmint.io/accept-credit-cards/integration-guides/polygon), or [Ethereum](https://docs.crossmint.io/accept-credit-cards/integration-guides/ethereum/add-crossmint-for-ethereum-react.js-next.js).
+### I'm using Payments
```
Comment (afeight) [inline packages/ui/react-ui/README.md:17]: I think these should match to what we have in docs, so here: "Add payment button to your site" as in https://docs.crossmint.com/docs/crossmint-pay-button

File: packages/ui/react-ui/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # `@crossmint/client-sdk-react-ui`
 
-## You can check the full documentation at [docs.crossmint.io](https://docs.crossmint.io/)
+## You can check the full documentation at [docs.crossmint.com](https://docs.crossmint.com/)
```
Comment (afeight) [inline packages/ui/react-ui/README.md:3]: nice

File: packages/ui/react-ui/README.md [hunk 3]
```diff
@@ -14,7 +14,20 @@ First, add the Crossmint Client SDK to your project with the following command:
...
 ```
 
-The next steps depend on which chain you're using for your project. For detailed integration instructions, visit our documentation for [Solana](https://docs.crossmint.io/accept-credit-cards/integration-guides/solana-candy-machine/b-i-have-an-existing-candy-machine-website/add-crossmint-for-solana-react.js-next.js), [Polygon](https://docs.crossmint.io/accept-credit-cards/integration-guides/polygon), or [Ethereum](https://docs.crossmint.io/accept-credit-cards/integration-guides/ethereum/add-crossmint-for-ethereum-react.js-next.js).
+### I'm using Payments
+
+The next steps depend on which chain you're using for your project. For detailed integration instructions, visit our documentation for [Solana](https://docs.crossmint.com/accept-credit-cards/integration-guides/solana-candy-machine/b-i-have-an-existing-candy-machine-website/add-crossmint-for-solana-react.js-next.js), [Polygon](https://docs.crossmint.com/accept-credit-cards/integration-guides/polygon), or [Ethereum](https://docs.crossmint.com/accept-credit-cards/integration-guides/ethereum/add-crossmint-for-ethereum-react.js-next.js).
+
+---
+
+### I'm using Wallets
```
Comment (afeight) [inline packages/ui/react-ui/README.md:23]: This one doesn't have an analogy on docs, so maybe "Display NFTs on your website"

File: packages/ui/react-ui/README.md [hunk 4]
```diff
@@ -14,7 +14,20 @@ First, add the Crossmint Client SDK to your project with the following command:
...
+
+We offer two components to help you quickly get up and running with your project using Crossmint wallets:
+
+-   `CrossmintNFTCollectionView` - Display a grid of NFTs.
+-   `CrossmintNFTDetail` - Display a card showing all NFT related details.
```
Comment (afeight) [inline packages/ui/react-ui/README.md:28]: screenshots of these in action would be great

**Comment by alfonso-paella** [comment]
Austin's comments sound sufficient to me.

Btw, no need to do tasks like these anymore. I tried feeding GPT4 our docs and the new API and it can do it really well

**Pull request merged**